autodistill/detection/detection_base_model.py

```diff
@@ -5,6 +5,7 @@
 
 import cv2
 import supervision as sv
+from roboflow import Roboflow
```

Comment:
I don't think roboflow is currently a dependency of `autodistill` so this'll break if we don't add it to the `requirements`